challenges/serializer-api.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+from .models import data
+from django.core.exceptions import ValidationError
+
+class RandnumbersSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = data
+        fields = ('id', 'numbers')
+
+    def validate_numbers(self, value):
+        queryset = data.objects.count()
+        if 500 != queryset:

```

Comment:
Where do these 500 numbers get sorted and stored once they are POSTed to the API?

Reply:
It looks like my models-api.py file didn't load into this pull request. The code for that file is:

from django.db import models

class data(models.Model):
    numbers = models.CharField(max_length=2000)

def __str__(self):
    return '%s %s' % (self.id, self.numbers)